### DISCUSSION TITLE
Windfloat Modeling and simulation

### DISCUSSION THREAD

**Discussion by sdu20951** [OpenFAST/openfast#1920] [Q&A] [answered]
I want to use openfast to calculate the motion response of the Windfloat wind turbine when there is no wind and waves. In the modeling process, the hydrodynamic files required by fast are calculated and converted by AQWA software. The files required for openfast are modified based on the 5MW_OC4Semi_WSt_WavesWN calculation example in the R-tset file provided on the official website. therein
1. Set the CompInflow marker as 0 in the fast file and modify the WtrDpth of the corresponding draft depth in ENVIRONMENTAL CONDITIONS.
![12343](https://github.com/OpenFAST/openfast/assets/107899938/0ec5a357-3598-4652-b05e-f72e5293d87e)
2. PtfmCMxt and PtfmCMzt in TURBINE CONFIGURATION are modified in ElastoDyn. PtfmMass, PtfmRIner, PtfmPIner, PtfmYIner in MASS AND INERTIA were also modified.
![111](https://github.com/OpenFAST/openfast/assets/107899938/baca4a79-9396-4083-a2e6-71caf02357bf)
![123](https://github.com/OpenFAST/openfast/assets/107899938/e10c2799-9ca8-4c44-b529-d0d58d399e9a)
3. Modified PtfmVol0 and PtfmCOBxt in HydroDyn FLOATING PLATFORM.
![1234](https://github.com/OpenFAST/openfast/assets/107899938/b3ddbbca-3102-4ba6-be1d-f935ba9a7d53)
Use additional damping instead of Morrison member
![surge](https://github.com/OpenFAST/openfast/assets/107899938/73cd5ad8-a87f-4e88-b201-a3966e907e59)
![11s1](https://github.com/OpenFAST/openfast/assets/107899938/a38805f5-4f26-498e-8b1d-886d276820d2)
4. The mooring system in MoorDyn is also set up accordingly.
The base of the wind turbine is at one end of WINDfloat. When preparing the hydrodynamic file of the platform, the center of gravity of the platform is calculated without considering the weight of the wind turbine.

But the results obtained by OPENFAST are completely different from those obtained by AQWA. What could possibly be the problem?
![surge](https://github.com/OpenFAST/openfast/assets/107899938/03acd5db-18c0-4442-95a7-a6dcc995a5f3)
![heave](https://github.com/OpenFAST/openfast/assets/107899938/da7ac0b1-77de-4091-9292-e60d4fb95f9a)
![pitch](https://github.com/OpenFAST/openfast/assets/107899938/5f700577-3fab-43f4-8f9b-ad05f2f2f9ce)

**Comment by jjonkman** [comment, marked as answer]
Dear @sdu20951,

Presumably in your graphs blue is OpenFAST and orange is AQWA?

Regardless--given that the WindFloat tower is not located at the center of the floater, but rather on an offset column--I'd expect that you'd also have to change the potential-flow solution (diffraction, radiation, hydrostatics).  I see that you are using your own WAMIT-formatted output files (_OOStarYangTest2655_), but you've kept `PtfmRefxt`=0, which implies that this potential-flow solution is relative to the tower centerline; is that correct?

I see a large mean heave in still water.  Does your full-system weight (rotor, drivetrain, nacelle, tower, floater) balance with the floater buoyancy and vertical mooring pretension?

When you eliminated the strip-theory solution from the orginal OC4-DeepCwind semisubmersible model, you also eliminated the water ballast.  Are you accounting for this ballast in the platform weight within ElastoDyn?

Best regards,

**Reply by sdu20951** [reply]
Thank you very much
The orange color is actually the Openfast result, and I think I've found the problem.
In OPENFAST, where is the average displacement of platform surge, heave and pitch movement? Is it (0,0,0)? Or at the center of gravity of the platform?

**Reply by jjonkman** [reply]
Dear @sdu20951,

There are multiple ways to output the platform motions.  Presuming you are outputting them from ElastoDyn (output parameters `PtfmSurge`, etc.), these are output relative to the ElastoDyn platform reference point, which in your model is (0,0,0) because `PtfmRefzt` = 0 m.

Best regards,

**Reply by sdu20951** [reply]
Dear 
Thank you very much
Best regards

**Comment by Tetsuya2000** [comment]
@sdu20951 did you get blue and orange lines to match, also what was it that triggered the problem, and what you do?
